phpunit.xml.dist

```diff
@@ -13,6 +13,9 @@
         >
 
     <testsuites>
+        <testsuite name="LexikJWTAuthenticationBundle Gen keys command">
+            <file>./Tests/Functional/Command/GenKeysCommandTest.php</file>
+        </testsuite>
```

Comment:
does not seem necessary, is it?

Reply:
It needed for this command test run before other tests.
It generate keys for other tests and there is no needed "in repo keys" for tests